src/scopes/trainingReports/collaborators.js

```diff
@@ -1,14 +1,18 @@
 import { filterAssociation } from './utils';
 
 const collaborators = `
-WITH unnested_collaborators AS (
+WITH
+unnested_collaborators AS (
     SELECT "id",unnest("collaboratorIds") FROM "EventReportPilots"
 )
 SELECT
  DISTINCT arr1."id"
-FROM "Users" "Users"
-    INNER JOIN unnested_collaborators arr1 ON arr1.unnest="Users"."id"
-WHERE "Users"."name"`;
+FROM "NationalCenters" "NationalCenters"
+INNER JOIN "NationalCenterUsers" "NationalCenterUsers"
+ON "NationalCenters".id = "NationalCenterUsers"."nationalCenterId"
+INNER JOIN unnested_collaborators arr1
+ON arr1.unnest="NationalCenterUsers"."userId"
+WHERE "NationalCenters"."name"`;

```

Comment:
Won't this search by National Center name (PFCE, for example) instead of collaborator name? I don't see this as part of the Jira.